tools/accuracy_checker/requirements-test.in

```diff
@@ -1,6 +1,7 @@
 pytest>=5.0,<=7.0.1; python_version < '3.10'
 pytest==7.2.0; python_version >= '3.10'
 pytest-mock~=2.0
+scipy>=1.5.4
```

Comment:
why??? it is requirements for accuracy checker unit tests, scipy do not used in tests

Reply:
I added it to eliminate one of PR check fail

Reply:
removed after making scipy optional.